### PR TITLE
Added Advance Alternate Solution

### PR DESCRIPTION
**Pull request by akmanon**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md [hunk 1]
```diff
@@ -164,6 +164,27 @@ uniteUnique([1, 3, 2], [5, 2, 1, 4], [2, 1]);
...
+    function uniteUnique(...arr) {
+    
+      return [...new Set(arr.flat())];;
+      
+    }
+
+    // test here
+    uniteUnique([1, 3, 2], [5, 2, 1, 4], [2, 1]);
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/@AshishKumar24/AdorableNeatCheckpoint' target='_blank' rel='nofollow'>Run Code</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md]: We're removing all external links like these from our repo. Please remove this

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md [hunk 2]
```diff
@@ -164,6 +164,27 @@ uniteUnique([1, 3, 2], [5, 2, 1, 4], [2, 1]);
...
+
+### Code Explanation:
+
+*   The spread operator `...` to create an array out of the Arguments object and flatten the array using Array.prototype.flat().
+*   We use Set data structure which store only uniques value and return the values of set as Array using spread operator `...` .
+
+#### Relevant Links
+*   <a href='https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flat' target='_blank' rel='nofollow'>Flat</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md]: Please change these links to markdown format

**Review by thecodingaviator — CHANGES_REQUESTED** [review]

File: guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md [hunk 3]
```diff
@@ -164,6 +164,27 @@ uniteUnique([1, 3, 2], [5, 2, 1, 4], [2, 1]);
...
+    function uniteUnique(...arr) {
+    
+      return [...new Set(arr.flat())];;
+      
+    }
+
+    // test here
+    uniteUnique([1, 3, 2], [5, 2, 1, 4], [2, 1]);
+
+![:rocket:](https://forum.freecodecamp.com/images/emoji/emoji_one/rocket.png?v=3 ":rocket:") <a href='https://repl.it/@AshishKumar24/AdorableNeatCheckpoint' target='_blank' rel='nofollow'>Run Code</a>
```
Comment (thecodingaviator) [inline guide/english/certifications/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/sorted-union/index.md]: I'm talking about this repl link when I refer to external links

**Comment by RandellDawson** [comment]
I propose adding your reference links in the Relevant Links section back in.

@thecodingaviator Instead of adding another solution, let's use this Alternative Advanced solution to replace the existing Alternative Advanced solution.  They are almost identical in logic.

**Comment by thecodingaviator** [comment]
@RandellDawson Sounds good. On a look, yes, this solution seems better

@akmanon Please go ahead and change the solution as @RandellDawson has requested above

**Comment by RandellDawson** [comment]
@akmanon Why did you close this PR?

**Pull request closed without merging**